addons/mrp/models/mrp_abstract_workorder.py

```diff
@@ -17,6 +17,7 @@ class MrpAbstractWorkorder(models.AbstractModel):
     product_uom_id = fields.Many2one('uom.uom', 'Unit of Measure', required=True, readonly=True)
     final_lot_id = fields.Many2one('stock.production.lot', string='Lot/Serial Number', domain="[('product_id', '=', product_id)]")
     product_tracking = fields.Selection(related="product_id.tracking")
+    consumption = fields.Selection(related="production_id.bom_id.consumption")
```

Comment:
it could be better that this field in dependant of the bom, (it is set during the workorder creation). like this if we change the consuption on the bom, on the already created workorder the beavior don't change.